docs/application-connector/docs/015-details-one-click-configuration.md

```diff
@@ -27,34 +27,47 @@ Follow these steps to configure the automatic connection between the Kyma Applic
 
 1. Get the configuration address URL with a valid token.
 
-Using the UI:
-
- - Go to the Kyma console UI.
- - Select **Administration**.
- - Select the **Remote Environments** from the **Integration** section.
- - Choose the Remote Environment to which you want to connect the external solution.
- - Click **Connect Remote Environment**.
- - Copy the token by clicking **Copy to clipboard**.
-
-Alternatively, get the configuration URL with a valid token using `kubectl port-forward` or `kubectl proxy`.
-
-  - Request:
-
-    First, run:
-    ```
-    kubectl -n=kyma-integration port-forward svc/connector-service-internal-api 8080:8080
-    ```
-    Send the request in a new terminal window:
-    ```
-    curl -X POST http://localhost:8080/v1/remoteenvironments/{remote-environment-name}/tokens
-    ```
-  - Response:
-    ```json
-    {
-        "url":"{CONFIGURATION_URL_WITH_TOKEN}",
-        "token":"example-token-123"
-    }
-    ```
+  Using the UI:
+
+   - Go to the Kyma console UI.
+   - Select **Administration**.
+   - Select the **Remote Environments** from the **Integration** section.
+   - Choose the Remote Environment to which you want to connect the external solution.
+   - Click **Connect Remote Environment**.
+   - Copy the token by clicking **Copy to clipboard**.
+
+  Alternatively, get the configuration URL with a valid token using `kubectl port-forward` or `kubectl proxy`.
+
+   - Request:
+
+      First, run:
+      ```
+      kubectl -n=kyma-integration port-forward svc/connector-service-internal-api 8080:8080
+      ```
+      Send the request in a new terminal window:
+      ```
+      curl -X POST http://localhost:8080/v1/remoteenvironments/{remote-environment-name}/tokens
+      ```
+   - Response:
+      ```json
+      {
+          "url":"{CONFIGURATION_URL_WITH_TOKEN}",
+          "token":"example-token-123"
+      }
+      ```
+  When you connect an external solution to a local Kyma deployment, you must set NodePort of the `core-nginx-ingress-controller` for the Gateway Service and for the Event Service.
```

Comment:
`you must set a NodePort` ?

Reply:
Yes, it's missing an article, but I'd say it should be "the".